v4/weaviate/data/objectsGetter.go

```diff
@@ -20,6 +20,10 @@ type ObjectsGetter struct {
 	additionalProperties []string
 	withLimit            bool
 	limit                int
+	withConsistencyLevel bool
+	consistencyLevel     string
+	withNodeName         bool
+	nodeName             string
```

Comment:
instead of using two additional bools, wouldn't `consistencyLevel/nodeName != "" ` equivalent?

Reply:
If you prefer I can do so, but just note that the rest of the client uses this pattern

Reply:
hmm, actually in this case we could rely on `consistencyLevel/nodeName != ""` check, those boolean flags were introduced to determine if the value was set, for example `limit int` field, with that type of var we need to have a `withLimit bool` to determine if the user for example explicitly have set `0` value

Reply:
but here with strings I think we can skip it

Reply:
removed them